go.mod

```diff
@@ -71,7 +71,10 @@ require (
 	github.com/robfig/cron/v3 v3.0.1
 	github.com/rs/zerolog v1.28.0
 	github.com/stretchr/testify v1.8.1
-	github.com/twmb/franz-go v1.9.0
+	github.com/twmb/franz-go v1.7.1
```

Comment:
please don't downgrade franz-go (how did this happen?)

Reply:
My vault, i have done a local downgrade because some dependencies cannot be downloaded there. I will upgrade to the right version.